### ISSUE TITLE
Question about training timestep setting in multi-reward training

### ISSUE THREAD

**Issue by MaoMao78777** [NVlabs/DiffusionNFT#44]
Hi! Thanks for your work.
In the paper, you mention using 40 rollout sampling steps for best visual quality in multi-reward training. So what is the number of training steps set in multi backward training? Is it 40 that is consistent with the number of sampling steps?

**Comment by ChenDRAG** [comment]
I think it is.  We use 40 steps by default if not comparing with FlowGRPO.  @zhengkw18 to confirm

**Comment by zhengkw18** [comment]
Yes, the training loss is computed on the same 40 timesteps.
